tempo_embeddings/text/passage.py

```diff
@@ -279,13 +305,20 @@ def from_text(
         window_size: int = None,
         window_overlap: int = None,
         metadata: dict = None,
+        nlp_pipeline = None,
     ) -> Iterable["Passage"]:
         """Create a Passage from a text string."""
 
         # TODO validate parameters
         # TODO: use window_size on tokens instead of characters
 
-        if window_size is None:
+        if nlp_pipeline is not None:
```

Comment:
As above, this if-else statement could be inverted to avoid the `if not ... else` logic.